src/views/Timeline.vue

```diff
@@ -34,16 +34,13 @@ export default {
       return _.sumBy(this.buckets, 'events.length');
     },
   },
-  watch: {
-    daterange() {
-      this.getBuckets();
-    },
-  },
   mounted: function() {
-    this.getBuckets();
+    this.daterange = [moment().subtract(this.timeintervalDefaultDuration, "seconds"), moment()],
```

Comment:
Why manually construct and set daterange here? Doesn't `input-timeinterval` set `this.daterange` correctly? (I guess it might not, since you removed the `v-model` behavior)